contentcuration/contentcuration/frontend/channelEdit/views/sync/SyncResourcesModal.vue

```diff
@@ -63,26 +79,40 @@
       :title="$tr('confirmSyncModalTitle')"
       :submitText="$tr('syncButtonLabel')"
       :cancelText="$tr('backButtonLabel')"
+      :size="600"
       @cancel="handleBack"
       @submit="handleSync"
     >
       <VSubheader>{{ $tr('confirmSyncModalExplainer') }}</VSubheader>
       <VCardText>
-        <ul class="mb-4 ml-3 mt-2">
-          <li v-if="syncFiles" class="font-weight-bold">
+        <ul class="mb-1 mt-1 no-bullets">
+          <li
+            v-if="syncFiles"
+            class="font-weight-bold"
+          >
             {{ $tr('syncFilesTitle') }}
           </li>
-          <li v-if="syncTags" class="font-weight-bold">
-            {{ $tr('syncTagsTitle') }}
+          <li
+            v-if="syncResourceDetails"
+            class="font-weight-bold"
+          >
+            {{ $tr('syncResourceDetailsTitle') }}
           </li>
-          <li v-if="syncTitlesAndDescriptions" class="font-weight-bold">
+          <li
+            v-if="syncTitlesAndDescriptions"
+            class="font-weight-bold"
+          >
             {{ $tr('syncTitlesAndDescriptionsTitle') }}
           </li>
-          <li v-if="syncExercises" class="font-weight-bold">
+          <li
+            v-if="syncExercises"
+            class="font-weight-bold"
+          >
             {{ $tr('syncExercisesTitle') }}
           </li>
         </ul>
       </VCardText>
+      <VSubheader>{{ $tr('confirmSyncModalWarningExplainer') }}</VSubheader>
```

Comment:
Lets make this a `<p>` too
```suggestion
      <p class="mb-0">{{ $tr('confirmSyncModalWarningExplainer') }}</p>
```